### DISCUSSION TITLE
Using the template without Docker

### DISCUSSION THREAD

**Discussion by giuseppericci** [fastapi/full-stack-fastapi-template#2138] [Questions] [answered]
I'm trying to use the FastApi template without Docker..So, I removed the useless files and the app it works..but it seems after user login it desn't recognize jwt token and for any operation (for example get user id) it replies with 403 Forbidden..Using the api acce token it works, but when I use test token it fails, it doesn't recognize the credentials and it seems it has different tokens.
Some help?
Thanks.

**Comment by GGSMK** [comment]
Did you find a solution?

**Comment by tiangolo** [comment, marked as answer]
If anyone is still having similar issues, please add a self contained minimal reproducible example so that I can replicate locally, that way I would be able to help.
